spec/models/observation_spec.rb

```diff
@@ -1532,61 +1534,59 @@
   end
 
   describe "species_guess parsing" do
-    before(:each) do
-      @observation = Observation.make!
-    end
-  
+    stub_elastic_index! Observation, Taxon
```

Comment:
Used where ES is not required in a larger spec where it is used (spec/models/observation_spec.rb:54)